build.sbt

```diff
@@ -6,7 +6,7 @@ import scalariform.formatter.preferences._
 val commonSettings = Seq(
   organization := "org.zalando",
   version := "0.2",
```

Comment:
I think version has to be set to "0.2.1" in that case otherwise it can break projects with "conflictManager := ConflictManager.strict" (haven't tested, but it happened to me in the past when I changed play / scala versions)